guides/shipments/shipping.md

```diff
@@ -1,129 +1,194 @@
-* Other attributes likely of interest to developers:
-  * `adjustment_total`
-  * `additional_tax_total`
-  * `promo_total`
-  * `included_tax_total`
-  * `cost`
-  * `order_id`
+### Basic attributes
 
-**Needed:** shipment process flow diagram
+- `number`: The unique identifier for this shipment. It begins with the letter
+  `H` and ends in an 11-digit number. This number is visible to customers, and
+  it can be used to find the order (by calling `Spree::Shipment.find_by(number:
+  H12345678910)`).
+- `tracking`: The identifier given for the shipping provider (such as FedEx or
+  UPS).
+- `shipped_at`: The time when the shipment is shipped.
+- `state`: The current state of the shipment.
+- `stock_location_id`: The ID of the stock location where the items for this
+  shipment are sourced from.
 
-An explanation of the different shipment states:
+### Additional attributes
 
-* `pending`: The shipment has backordered inventory units and/or the order is not paid for.
-* `ready`: The shipment has *no* backordered inventory units and the order is paid for.
-* `shipped`: The shipment is on its way to the buyer.
-* `canceled`: When an order is cancelled, all of its shipments will also be cancelled. When this happens, all items in the shipment will be restocked. If an order is "resumed", then the shipment will also be resumed.
+In addition to the basic attributes, developers may be interested in the
+following shipment attributes: 
 
-Explaining each piece of the shipment world inside of Solidus separately and how each piece fits together can be a cumbersome task. Fortunately, using a few simple examples makes it much easier to grasp. In that spirit, the examples are shown first in this guide.
+- `adjustment_total`: The sum of the promotion and tax adjustments on the
+  shipment. 
+- `additional_tax_total`: The sum of U.S.-style sales taxes on the shipment.
+- `promo_total`: The sum of the promotions on the shipment. 
+- `included_tax_total`: The sum of the VAT-style taxes on the shipment. 
+- `cost`: The order cost for the select shipping method.
+- `order_id`: The ID for the order that a shipment is being created for.
 
-## Definitions, Design, & Functionality
+<!-- TODO:
+  Add a shipment process flow diagram.
+-->
 
-To properly leverage Solidus' shipping system's flexibility you must understand a few key concepts:
+### Shipping states
 
-* Stock Locations
-* Shipping Methods
-* Zones
-* Shipping Categories
-* Calculators (through Shipping Rates)
+Each shipment is assigned a `state` attribute. Depending on its state, different
+actions can be performed on shipments. There are four possible states:
 
-### Stock Locations
-Stock locations represent physical storage locations from which stock is shipped.
+- `pending`: The shipment has backordered inventory units and/or the order is
+  not paid for.
+- `ready`: The shipment has no backordered inventory units and the order is paid
+  for.
+- `shipped`: The shipment is on its way to the buyer.
```

Comment:
The shipment has left the warehouse. 

If it has arrived at the buyer, it will still be shipped, but not "on its way" anymore.